review/rhea-promise.api.md

```diff
@@ -0,0 +1,589 @@
+## API Report File for "rhea-promise"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+/// <reference types="node" />
+
+import { AmqpError } from 'rhea';
+import { Connection as Connection_2 } from 'rhea';
```

Comment:
is it possible to avoid these?

Reply:
It's a known api-extractor issue when there are types with the same names. I can live with it even it is not great.